src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -163,6 +158,12 @@ export class FilterTimestamp extends HTMLElement {
   }
 
   disconnectedCallback() {
+    // context menu doesn't work on devtool_page
+    // see: https://github.com/mozilla/extension-activity-monitor/issues/43
+    if (!browser.menus?.onClicked || !browser.menus?.onHidden) {
+      return;
+    }
+
```

Comment:
@atiqueahmedziad it may be better to do the opposite here and change the two calls to `browser.menus` to make sure using the optional chaining operator so that they will become a no-op if `browser.menus` isn't available. I think this is less risky because by returning earlier we may mistakenly skip also some other call that should instead be called (e.g. in this particular case we may still want to call `this.filterToggleBar.removeEvnetListener` and `this.dropDownList.removeEventListener`).

It should look something like:
```
    browser.menus?.onClicked.removeListener(this.setFilterRange);
    browser.menus?.onHidden.removeListener(this.onHiddenListener);
```

Reply:
I agree. :-) Thanks for the suggestion